fiatconnect-api.md

```diff
@@ -1891,15 +1938,18 @@ one returned from the `GET /transfer/:transferId/status` endpoint.
 	fee?: `string`,
 	fiatAccountId: `string`,
 	transferId: `string`,
-	transferAddress: `string`
-	txHash?: `string`
+	transferAddress: `string`,
+	txHash?: `string`,
+	userActionDetails?: `TransferInUserActionDetailsEnum`
 }
 ```
 
 ### 5.1.3. `WebhookTransferOutStatusEventSchema`
 
 `WebhookTransferOutStatusEventSchema` is the schema that defines webhook payloads for transfer out events. Note that this schema is nearly identical to the
-one returned from the `GET /transfer/:transferId/status` endpoint; it lacks the optional `txHash` field, since it is not relevenat for transfers out.
+one returned from the `GET /transfer/:transferId/status` endpoint; it lacks the optional `txHash`, `userActionType` and `userActionDetails` field,
+since these are not relevant to transfers out. The semantics for fields and their inclusion are otherwise identical to those of the `GET /transfer:transferId/status`
```

Comment:
```suggestion
since these are not relevant to transfers out. The semantics for fields and their inclusion are otherwise identical to those of the `GET /transfer/:transferId/status`
```